diff --git a/node_modules/js-pixel-fonts/index.js b/node_modules/js-pixel-fonts/index.js
index 4c16e98..d882c58 100644
--- a/node_modules/js-pixel-fonts/index.js
+++ b/node_modules/js-pixel-fonts/index.js
@@ -11,7 +11,7 @@ const gap = [[0]];
 const areTouching = (first, second) => {
     for (let i = 0; i < first.length; ++i) {
         if (first[i] && first[i][first[i].length - 1] === 1) {
-            for (j = -1; j <= 1; ++j) {
+            for (let j = -1; j <= 1; ++j) {
                 if (second[i+j] && second[i+j][0] === 1) {
                     return true;
                 }
